src/Tools/AI Test Toolkit/src/TestSuite/AITBatchRunDialog.Page.al

```diff
@@ -0,0 +1,45 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+page 149043 "AIT Batch Run Dialog"
+{
+    Caption = 'Batch Run';
+    PageType = StandardDialog;
+    ApplicationArea = All;
+    UsageCategory = None;
+
+    layout
+    {
+        area(content)
+        {
+            group(Control1)
+            {
+                ShowCaption = false;
+
+                field(Iterations; Iterations)
+                {
+                    ApplicationArea = All;
+                    Caption = 'Number of iterations';
+                    ToolTip = 'Specifies the number of iterations to run.';
+                }
+            }
+        }
+    }
+
+    var
+        Iterations: Integer;
```

Comment:
How is this incremented?

Reply:
It's set on the page. Default is 1

Reply:
So we cant have more than 1?

Reply:
Yes, iterations is a field on the page, you can modify it. See the field "Iterations" above. When you open the page, it's set to 1, then you can change it to the number of iterations that you want to run.